pkg/hintrunner/zero/zerohint_uint256.go

```diff
@@ -223,3 +223,99 @@ func createUint256SignedNNHinter(resolver hintReferenceResolver) (hinter.Hinter,
 	}
 	return newUint256SignedNNHint(a), nil
 }
+
+func newUint256UnsignedDivRemHint(a, div, quotient, remainder hinter.ResOperander) hinter.Hinter {
+	return &GenericZeroHinter{
+		Name: "Uint256UnsignedDivRem",
+		Op: func(vm *VM.VirtualMachine, _ *hinter.HintRunnerContext) error {
+			//> a = (ids.a.high << 128) + ids.a.low
+			//div = (ids.div.high << 128) + ids.div.low
+			//quotient, remainder = divmod(a, div)
+			//ids.quotient.low = quotient & ((1 << 128) - 1)
+			//ids.quotient.high = quotient >> 128
+			//ids.remainder.low = remainder & ((1 << 128) - 1)
+			//ids.remainder.high = remainder >> 128
+			aLow, aHigh, err := GetUint256AsFelts(vm, a)
+			if err != nil {
+				return err
+			}
+			var aLowBig big.Int
+			aLow.BigInt(&aLowBig)
+			var aHighBig big.Int
+			aHigh.BigInt(&aHighBig)
+
+			divLow, divHigh, err := GetUint256AsFelts(vm, div)
+			if err != nil {
+				return err
+			}
+			var divLowBig big.Int
+			divLow.BigInt(&divLowBig)
+			var divHighBig big.Int
+			divHigh.BigInt(&divHighBig)
+
+			a := new(big.Int).Add(new(big.Int).Lsh(&aHighBig, 128), &aLowBig)
+			div := new(big.Int).Add(new(big.Int).Lsh(&divHighBig, 128), &divLowBig)
+			quot, rem := utils.FeltDivRem(new(fp.Element).SetBigInt(a), new(fp.Element).SetBigInt(div))
+
+			var quotBig big.Int
+			quot.BigInt(&quotBig)
+
+			var remBig big.Int
+			rem.BigInt(&remBig)
+
+			mask := new(big.Int).Sub(new(big.Int).Lsh(big.NewInt(1), 128), big.NewInt(1))
+
+			lowQuot := new(fp.Element).SetBigInt(new(big.Int).And(&quotBig, mask))
+			highQuot := new(fp.Element).SetBigInt(new(big.Int).Rsh(&quotBig, 128))
+
+			lowRem := new(fp.Element).SetBigInt(new(big.Int).And(&remBig, mask))
+			highRem := new(fp.Element).SetBigInt(new(big.Int).Rsh(&remBig, 128))
+
+			quotientAddr, err := quotient.GetAddress(vm)
+			if err != nil {
+				return err
+			}
+			x := memory.MemoryValueFromFieldElement(lowQuot)
+			err = vm.Memory.WriteToAddress(&quotientAddr, &x)
+			if err != nil {
+				return err
+			}
+			err = hinter.WriteToNthStructField(vm, quotient, memory.MemoryValueFromFieldElement(highQuot), 1)
+			if err != nil {
+				return err
+			}
```

Comment:
You've had to use this logic several times, I'd suggest you create a utility for uint256 values, something like `WriteUint256ToAddress()`, that does this